internal/server/server_test.go

```diff
@@ -1,28 +1,23 @@
 package server
 
 import (
+	"net/http"
 	"net/http/httptest"
+	"os"
 	"testing"
-)
-
-func TestHandleRoot(t *testing.T) {
-	tests := []struct {
-		method string
-		path   string
-	}{
-		{method: "GET", path: "/"},
-	}
 
-	s := New()
+	"github.com/stretchr/testify/assert"
+)
 
-	for _, test := range tests {
-		req := httptest.NewRequest(test.method, test.path, nil)
-		w := httptest.NewRecorder()
+var s http.Handler
 
-		s.ServeHTTP(w, req)
+func TestMain(m *testing.M) {
+	s = New()
+	os.Exit(m.Run())
+}
 
-		if w.Code != 200 {
-			t.Error("test failed")
-		}
-	}
+func TestHandleRoot(t *testing.T) {
+	w := httptest.NewRecorder()
+	s.ServeHTTP(w, httptest.NewRequest("GET", "/", nil))
+	assert.Equal(t, http.StatusOK, w.Code)
```

Comment:
Is the plan to expand these tests as new features are ported over from the original auth portal?

Reply:
Yes, new tests can be added as the routes are updated.